config/env/default.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
   },
   // sessionSecret should be changed for security measures and concerns
   sessionSecret: process.env.SESSION_SECRET || 'MEAN',
-  // sessionKey is set to the generic sessionId key used by PHP applications
-  // for obsecurity reasons
+  // sessionKey is the session name and is set to the generic sessionId key
+  // used by PHP applications for obsecurity reasons
```

Comment:
I'm totally puzzled by it. :-) This config var sets the name for the session cookie, right?

So I suppose it's not referring to PHP, the language?

Also first time seeing used "obsecurity" when saying "security by obscurity", but I like it!